example/thirdparty/arrow/build.mill

```diff
@@ -133,11 +127,11 @@ object `package` extends RootModule {
       s"-Xcommon-sources=${files.mkString(",")}"
     }
 
-    trait ArrowPlatformModule extends KotlinModule {
+    trait ArrowPlatformModule extends KotlinMavenModule {  outer =>
```

Comment:
`KotlinMavenModule` implies that layout is `src/main/...` while for multiplatform setup it is usually `src/commonMain/... ` + `src/<target>Main/...`. What is the advantage of applying it?

Reply:
Good question. I think it's pretty arbitrary, but it feels closer than the raw `src/` folder that `KotlinModule` provides, and at least it puts the `src/main/resources` and `src/test/resources` in the right place. I also ended up using it in `ArrowJvmModule` which has the exact correct layout, and so thought it would be nice to be consistent even if it doesn't quite fit